fighters/robot/src/acmd/aerials.rs

```diff
@@ -222,65 +227,47 @@ unsafe fn robot_attack_air_b_game(fighter: &mut L2CAgentBase) {
                 WorkModule::set_float(boma, current_fuel - boost_fuel_depletion, *FIGHTER_ROBOT_INSTANCE_WORK_ID_FLOAT_BURNER_ENERGY_VALUE);
             }
         }
+		
+		if VarModule::is_flag(fighter.battle_object, vars::robot::status::BOOST_ATTACK){
+			FT_MOTION_RATE(fighter, 10.0/(8.0-3.0));
+		} else {
+			FT_MOTION_RATE(fighter, 0.7);
```

Comment:
please use `FT_MOTION_RATE(fighter, num_frames / (end_frame - curr_frame))` for line 234, like on line 232

- end_frame is the frame you change motion rate next (or the last frame of the move)
- curr_frame is the frame you're writing this line on
- num_frames is the new number of frames you would like to get from curr_frame to end_frame